### PR TITLE
Update integration tests and semantic updates

### PR DESCRIPTION
**Pull request by ehdsouza**
This PR does the following:
* Updates integration tests to RC instances
* Includes examples for the ones which don't have integration tests

Also, @padamstx could you review the semantic release updates?

### CODE REVIEW

File: .releaserc [hunk 1]
```diff
@@ -1,15 +1,18 @@
 {
   "branch": "master",
-  "verifyConditions": [],
+  "verifyConditions": ["@semantic-release/changelog", "@semantic-release/github"],
+  "debug": true,
```
Comment (padamstx) [inline .releaserc]: probably don't need the "debug" field here, but also ok to leave it you want... I think it just results in more output in the travis build log

File: .releaserc [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
   "prepare": [
+    "@semantic-release/changelog",
```
Comment (padamstx) [inline .releaserc]: the "exec" step needs to be done first, then changelog, then git.   You can see the python-core file as an example.

**Comment by germanattanasio** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/integration/test_examples.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/watson-developer-cloud/python-sdk/pullRequest?prid=4467801&bid=15087711)

**Comment by germanattanasio** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity decreasing per file
==============================
+ test/integration/test_examples.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/watson-developer-cloud/python-sdk/pullRequest?prid=4467801&bid=15087711)

**Pull request merged**